Makefile

```diff
@@ -85,37 +97,28 @@ gen/.build:
 
 base.c: $(CONF)
 
-$(DEPS): $(SRCS)
```

Comment:
Could you let me know why you think this part should be removed? Have you verified buildings on other platforms like Linux, BSD and OpenWRT? I have no confidence to merge your changes due to removal of these lines.